apps/extension/src/routes/page/onboarding/routes.tsx

```diff
@@ -6,7 +6,7 @@ import { ImportSeedPhrase } from './import';
 import { OnboardingSuccess } from './success';
 import { SetPassword } from './set-password';
 import { pageIndexLoader } from '..';
-import { SetRpcEndpoint } from './set-rpc-endpoint';
+import { SetGrpcEndpoint } from './set-grpc-endpoint';
```

Comment:
Consolidating the naming

Reply:
would prefer to align on "rpc", it's a more generic term

Reply:
Ugh, same. I'll merge this and then discuss in Discord. The issue was, the local storage key is `grpcEndpoint`, so I figured we should standardize on that since it's already in a bunch of users' local storage.